### PR TITLE
Publish artifacts required for integrity checking tools

### PR DESCRIPTION
**Pull request by miklos-da**
Summary of changes:
* Moved actual command-line drivers to package `kvutils.tools.driver`.
* Moved export/integrity checking supporting classes to package `kvutils.tools.export`.
* Publish integrity checking related classes in artifacts `data-export-kvutils`.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/participant-state/kvutils/tools/BUILD.bazel [hunk 1]
```diff
@@ -5,13 +5,37 @@ load(
...
+        "@maven//:com_typesafe_akka_akka_stream_2_12",
+        "@maven//:io_dropwizard_metrics_metrics_core",
+    ],
+)
+
 da_scala_binary(
     name = "integrity-check",
-    srcs = ["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/IntegrityCheck.scala"],
-    main_class = "com.daml.ledger.participant.state.kvutils.tools.IntegrityCheck",
+    srcs = ["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/driver/IntegrityCheck.scala"],
```
Comment (SamirTalwar-DA) [inline ledger/participant-state/kvutils/tools/BUILD.bazel:37]: Why do you need the `srcs` section at all? It looks like this is included by the `:tools` library.

File: ledger/participant-state/kvutils/tools/BUILD.bazel [hunk 2]
```diff
@@ -34,8 +58,8 @@ da_scala_binary(
 
 da_scala_binary(
     name = "integrity-check-v2",
-    srcs = glob(["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/*"]),
-    main_class = "com.daml.ledger.participant.state.kvutils.tools.IntegrityCheckV2",
+    srcs = glob(["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/driver/IntegrityCheckV2.scala"]),
```
Comment (SamirTalwar-DA) [inline ledger/participant-state/kvutils/tools/BUILD.bazel:61]: Here too.

File: ledger/participant-state/kvutils/tools/BUILD.bazel [hunk 3]
```diff
@@ -5,13 +5,37 @@ load(
...
+        "@maven//:io_dropwizard_metrics_metrics_core",
+    ],
+)
+
 da_scala_binary(
     name = "integrity-check",
-    srcs = ["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/IntegrityCheck.scala"],
-    main_class = "com.daml.ledger.participant.state.kvutils.tools.IntegrityCheck",
+    srcs = ["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/driver/IntegrityCheck.scala"],
+    main_class = "com.daml.ledger.participant.state.kvutils.tools.driver.IntegrityCheck",
```
Comment (fabiotudone-da) [inline ledger/participant-state/kvutils/tools/BUILD.bazel:38]: Why do we need to keep V1?

File: ledger/participant-state/kvutils/tools/BUILD.bazel (hunk 1 above)
Comment (miklos-da) [reply ledger/participant-state/kvutils/tools/BUILD.bazel]: Good point -- removed.

File: ledger/participant-state/kvutils/tools/BUILD.bazel (hunk 2 above)
Comment (miklos-da) [reply ledger/participant-state/kvutils/tools/BUILD.bazel]: Removed.

File: ledger/participant-state/kvutils/tools/BUILD.bazel (hunk 3 above)
Comment (miklos-da) [reply ledger/participant-state/kvutils/tools/BUILD.bazel:37]: For backwards compatibility reasons. It'll make sense to phase out v1 once we start publishing ledger exports in the new format and all build tools will be using v2.

**Pull request merged**